docs/build/mapping/store.md

```diff
@@ -0,0 +1,128 @@
+# Advanced Access to the SubQuery Store
+
+The SubQuery store is an injected class that allows users to interact with record in the database from within mapping functions. This will come handy when user demand use multiple entity records as the parameters in the mapping function, or create/update multiple records in a single place.
```

Comment:
```suggestion
The SubQuery store is an injected class that allows users to interact with records in the database from within mapping functions. This will come handy when user demands using multiple entity records as the parameters in the mapping function, or create/update multiple records in a single place.
```